.github/workflows/pr_tests.yml

```diff
@@ -12,6 +12,16 @@ jobs:
       - name: Run ruff
         run: ruff check space_packet_parser
 
+  ast:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install bandit
+        run: pip install . bandit
```

Comment:
We should add the Ruff checks you want rather than adding another linter: https://docs.astral.sh/ruff/rules/#flake8-bandit-s

Reply:
Oh wow, Ruff is amazing. If I'm understanding the config correctly, you've already enabled Bandit checking with `lint.select = [..., "S", ...]`. Is that right?

Reply:
Yes, I believe so. But they may have slightly different enable/disable settings or not implemented yet. Yeah, Ruff has subsumed many of the disjointed linter packages, so more of a one-stop-shop.